client/src/components/ListItems.tsx

```diff
@@ -9,7 +9,7 @@ const ListItems = ({ value, forceList }: ListItemsProps) => {
   const items = value.split(/[\r\n\v\f\u2028\u2029]+/)
   if (items.length > 1 || forceList) {
     return (
-      <ul>
+      <ul style={{ marginLeft: -18 }}>
```

Comment:
Nno-zero length values must specify a unit, so this should probably be `marginLeft: "-18px"` ?